docs/3-reference/useCheckoutPricingInput.stories.mdx

```diff
@@ -0,0 +1,217 @@
+import { Subtitle } from '@storybook/addon-docs/blocks';
+import { PropsTable } from '@storybook/components';
+
+<Meta title="Reference/UseCheckoutPricingInput" />
+
+# UseCheckoutPricingInput
+
+<Subtitle slot={() => 'To interact with useCheckoutPricing'} />
+
+A plain object used to set initial values for [useCheckoutPricing][use-checkout-pricing-input] and to update those values if needed with [setCheckoutPricing][use-checkout-pricing-set-pricing].
```

Comment:
Thoughts on adding a note that overall usage is covered in the `useCheckoutPricing` section?